dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/dao/SimpleMongoDBStorageStrategy.java

```diff
@@ -53,8 +54,11 @@
     public static final String OBJECT_TYPE_VALUE = XMLSchema.ANYURI.stringValue();
     public static final String CONTEXT = "context";
     public static final String PREDICATE = "predicate";
-    public static final String OBJECT = "object";
+    public static final String PREDICATE_HASH = "predicate_hash";
+    public static final String OBJECT = "object_original";
```

Comment:
Can you change this to just "object" or change "context" "predicate" "subject" to "xxx_original"